### PR TITLE
Preparing for Brazilian

### PR DESCRIPTION
**Pull request by alemorvan**
We have to prepare mkdocs for Brazilian language.
This time, language code will be with 5 letters code.

I think, it will come from crowdin with pt-BR (uppercase).

Please check if crowdin can support both 2 letters codes and 5 letters codes.

Antoine.

### CODE REVIEW

**Comment by alemorvan** [comment]
@NeilHanlon please check MM (~localization) for explaination.
Thx!

**Comment by sspencerwire** [comment]
> @NeilHanlon please check MM (~localization) for explaination. Thx!

@NeilHanlon @alemorvan Turns out that this **will not** work the way it is entered. Mkdocs explodes. It needs to be "pt_BR" rather than "pt-BR" the mapping has been fixed in Crowdin, but we need to correct this PR before merging.

File: mkdocs.yml [hunk 1]
```diff
@@ -74,6 +74,7 @@ plugins:
           sv: Svenska
           pl: Polish
           pt: Português
+          pt-BR: Brazilian
```
Comment (alemorvan) [inline mkdocs.yml:77]:
```suggestion
          pt_BR: Brazilian
```

File: mkdocs.yml [hunk 2]
```diff
@@ -110,6 +111,8 @@ extra:
       lang: pl
     - name: Português
       link: ./pt/
+    - name: Brazilian
+      link: ./pt-BR/
```
Comment (alemorvan) [inline mkdocs.yml:115]:
```suggestion
      link: ./pt_BR/
```

**Comment by sspencerwire** [comment]
This works in my local copy of `mkdocs` so it should now be ready to merge. @ressonix can you merge this change for us?

**Review by NeilHanlon — APPROVED** [review]

**Comment by NeilHanlon** [comment]
Yeah this looks good to me. I can merge but I would like @rocky-linux/web-leads to review so everyone knows what's up :)

**Pull request merged**